src/burp/restore.epp

```diff
@@ -421,7 +421,7 @@ int RESTORE_restore (const TEXT* file_name, const TEXT* database_name)
 		// Block added to verbose index creation by Toni Martir
 		// Always try to activate deferred indices - it helps for some broken backups,
 		// and in normal cases doesn't take much time to look for such indices. AP-2008.
-		EXEC SQL SET TRANSACTION ISOLATION LEVEL READ COMMITTED NO_AUTO_UNDO;
+		EXEC SQL SET TRANSACTION ISOLATION LEVEL READ COMMITTED NO_AUTO_UNDO AUTO_RELEASE_TEMP_BLOBID;
```

Comment:
Is it really required for every transaction in restore.epp ? 

For example, if gbak run against pre-FB6 server, this will fail (for every transaction!) and gbak will start every transaction second time and without NO_AUTO_UNDO also. Is it good ?

Another way would be to remember at first failure that target server not supported AUTO_RELEASE_TEMP_BLOBID or NO_AUTO_UNDO and don't ever try to start transaction with unsupported options. 
Though, such change it out of scope of this PR...

Reply:
> Is it really required for every transaction in restore.epp ?

It's not really. But it's not so obvious to determine particular places where the new transaction option should be used. Ideally, this needs to be tested carefully (also considering gbak options which affect commit points) to make sure there are no missed places. So now I think it would be better to set transaction options at one place. Even if AUTO_RELEASE_TEMP_BLOBID is not going to make a difference for some transaction, it won't harm anyway. And I decided to solve this problem using the approach you suggested.

Reply:
Thanks!